core/src/Cabana_Halo.hpp

```diff
@@ -99,22 +102,20 @@ class Halo : public CommunicationPlan<MemorySpace>
       \note Calling this function completely updates the state of this object
```

Comment:
This seems like an odd thing to say here; can you remove these original notes and from your new constructors?